lib/m2ts/caption-stream.js

```diff
@@ -382,18 +382,20 @@ Cea608Stream.prototype = new Stream();
 // Trigger a cue point that captures the current state of the
 // display buffer
 Cea608Stream.prototype.flushDisplayed = function(pts) {
-  var row, i;
-
-  for (i = 0; i < this.displayed_.length; i++) {
-    row = this.displayed_[i];
-    if (row.length) {
-      this.trigger('data', {
-        startPts: this.startPts_,
-        endPts: pts,
-        // remove spaces from the start and end of the string
-        text: row.trim()
-      });
-    }
+  var content = this.displayed_
+    // remove spaces from the start and end of the string
+    .map(function(row){ return row.trim(); })
```

Comment:
We don't have vjsstandard running on this project but if we did, it would complain about the missing space after the closing parens in the function definition.